src/mococrw/asymmetric_crypto_ctx.h

```diff
@@ -477,6 +483,17 @@ class ECDSASignaturePublicKeyCtx : public DigestVerificationCtx, public MessageV
     ECDSASignaturePublicKeyCtx(const AsymmetricPublicKey& key,
                                openssl::DigestTypes hashFunction = openssl::DigestTypes::SHA256);
 
+    /**
+     * @brief Constructor
+     *
+     * @param key The public key to be used
+     * @param sigFormat The format that in which signatures are provided
```

Comment:
Doxygen params are in the wrong order, same with the `X509Certificate` version below